### PR TITLE
Adding ci and release files

### PR DESCRIPTION
**Pull request by hcherukuri**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Release collection
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
```
Comment (rpelisse) [inline .github/workflows/release.yml:10]: @sabre1041 @hcherukuri I suppose we can't run on Fedora?

**Comment by rpelisse** [comment]
LGTM... @sabre1041 any comment?

File: .github/workflows/release.yml (hunk 1 above)
Comment (hcherukuri) [reply .github/workflows/release.yml:10]: I am not sure,  Need to check or Andrew might know.

Comment (rpelisse) [reply .github/workflows/release.yml:10]: Neh, forget it. I recall Andy and I already discussed that, I think there is, indeed, no other choice.

**Comment by sabre1041** [comment]
um. theres molecule tests. yet no molecule code?????

**Comment by hcherukuri** [comment]
I have removed the molecule tests block from the CI.yml

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+name: CI
+"on":
+  push:
+    branches:
+      - master
```
Comment (sabre1041) [inline .github/workflows/ci.yml:6]: Change to `main` branch

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  pull_request:
+
+jobs:
+  molecule:
```
Comment (sabre1041) [inline .github/workflows/ci.yml:10]: Change job name from `molecule` to `ci`

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        uses: actions/checkout@v2
+        with:
+          path: ansible_collections/middleware_automation/redhat_csp_download
+
+      - name: Set up Python ${{ matrix.python_version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python_version }}
+
+      - name: Install molecule
```
Comment (sabre1041) [inline .github/workflows/ci.yml:26]: We do not need to install `molecule` anymore. Step title should be changed and remove the pip module

**Review by sabre1041 — CHANGES_REQUESTED** [review]

**Comment by rpelisse** [comment]
@hcherukuri did you see the last comment of Andrew?

**Comment by hcherukuri** [comment]
> @hcherukuri did you see the last comment of Andrew?

I am just checking into it.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**